include/swift/AST/diagnostics/en.yaml

```diff
@@ -6701,9 +6701,13 @@
     %select{associated value|stored property}0 type %1 does not conform to
     protocol %2, preventing synthesized conformance of %3 to %2
 
-- id: classes_automatic_protocol_synthesis
+- id: automatic_protocol_synthesis_unsupported
   msg: >-
-    automatic synthesis of '%0' is not supported for classes
+    automatic synthesis of '%0' is not supported for %select{classes|structs}1
+
+- id: comparable_synthesis_raw_value_not_allowed
+  msg: >-
+    "enum declares raw type %0, preventing synthesized conformance of %1 to %2"
```

Comment:
I don't think adding double quotations for multi-line YAML strings would work. Would be great if you can just remove the quotation.

Reply:
Good point. Either remove `>-` or quotations :)

Reply:
Yes either way would work. But I think if all messages had the same format i.e. `>-` it will look better (because all of the messages has the same style), of course, it's not that big of a deal. :)